### ISSUE TITLE
[FEATURE REQUEST] Quick Buy Button

### ISSUE THREAD

**Issue by xoxfaby** [IsThereAnyDeal/AugmentedSteam#1399]
### Desired Behavior 

Add a button to steam market pages for items to "Quick Buy" and "Insta Buy". These buttons would quickly make an order to either put up a buy order a +0.01 or buy the lowest sell order, respectively. 

### Motivation / Use Case for Changing the Behavior

Same reason as sell buttons, just saves time.

**Comment by candela97** [comment]
This feature should be enabled by default already, make sure you have the corresponding option under the Inventory section checked.

**Comment by xoxfaby** [comment]
For sell buttons, not "Buy" buttons

**Comment by candela97** [comment]
Oh sorry I misread the issue, there's no quick purchase feature. I've found similar suggestions made to the original ES, but none of them received a response. I think this is doable, but we'll probably need to add a way to prevent missclicks.

**Comment by xoxfaby** [comment]
Yeah, makes sense. I think it would be good to have these buttons on most of not all places where you can see items, I can see use cases for all I can think of, especially the market search so you can search for a set of things and quickly buy them all.

**Comment by avi12** [comment]
Such a Quick Buy button should only be visible if the Steam Wallet has enough money; otherwise, attempting to automatically add funds might not work and the payment method that the user typically uses could return an error
